cleanlab/datalab/data_issues.py

```diff
@@ -22,16 +22,17 @@
 instances and keeps track of each issue, a summary for each type of issue,
```

Comment:
share PDF showing what the output looks like when somebody runs all of the following:

- datalab.find_issues()
- datalab.report()
- datalab.issue_summary
- datalab.issues
- datalab.info
- datalab.get_issues(SPECIFIC_IMAGE_ISSUE)
- datalab.get_summary(SPECIFIC_IMAGE_ISSUE)
- datalab.get_info(SPECIFIC_IMAGE_ISSUE)